tests/perfalert/test_auto_perf_sheriffing/conftest.py

```diff
@@ -86,6 +87,17 @@ def record_ready_for_processing(linux_perf_alert, record_context_sample):
     return record
 
 
+@pytest.fixture
+def record_from_mature_report(test_perf_alert_2):
+    # create a record from a mature report
+    date_past = datetime.utcnow() - timedelta(hours=10)
+    with patch('django.utils.timezone.now', Mock(return_value=date_past)):
```

Comment:
```suggestion
    from freezegun import freeze_time
    with freeze_time(date_past):
```

Reply:
You can inline `date_past`' s value. This alternate solution has the advantage of not being too verbose on what must be mocked. It does it automatically & it's probably more future proof.